auxx/src/Command/Tx.hs

```diff
@@ -212,36 +214,58 @@ newtype AuxxException = AuxxException Text
 
 send
-    :: forall m. MonadAuxxMode m
+    :: forall m
+     . MonadAuxxMode m
     => ProtocolMagic
+    -> SlotCount
     -> Diffusion m
     -> Int
     -> NonEmpty TxOut
     -> m ()
-send pm diffusion idx outputs = do
+send pm epochSlots diffusion idx outputs = do
     skey <- takeSecret
     let curPk = encToPublic skey
-    let plainAddresses = map (flip makePubKeyAddress curPk . IsBootstrapEraAddr) [False, True]
-    let (hdAddresses, hdSecrets) = unzip $ map
-            (\ibea -> fromMaybe (error "send: pass mismatch") $
-                    deriveFirstHDAddress (IsBootstrapEraAddr ibea) emptyPassphrase skey) [False, True]
+    let plainAddresses = map
+            (flip makePubKeyAddress curPk . IsBootstrapEraAddr)
+            [False, True]
+    let
+        (hdAddresses, hdSecrets) = unzip $ map
+            (\ibea ->
+                fromMaybe (error "send: pass mismatch") $ deriveFirstHDAddress
+                    (IsBootstrapEraAddr ibea)
+                    emptyPassphrase
+                    skey
+            )
+            [False, True]
     let allAddresses = hdAddresses ++ plainAddresses
-    let allSecrets = hdSecrets ++ [skey, skey]
-    etx <- withSafeSigners allSecrets (pure emptyPassphrase) $ \signers -> runExceptT @AuxxException $ do
-        let addrSig = HM.fromList $ zip allAddresses signers
-        let getSigner addr = HM.lookup addr addrSig
-        -- BE CAREFUL: We create remain address using our pk, wallet doesn't show such addresses
-        (txAux,_) <- lift $ prepareMTx pm getSigner mempty def (NE.fromList allAddresses) (map TxOutAux outputs) curPk
-        txAux <$ (ExceptT $ try $ submitTxRaw diffusion txAux)
+    let allSecrets   = hdSecrets ++ [skey, skey]
+    etx <- withSafeSigners allSecrets (pure emptyPassphrase) $ \signers ->
+        runExceptT @AuxxException $ do
+            let addrSig = HM.fromList $ zip allAddresses signers
+            let getSigner addr = HM.lookup addr addrSig
+            -- BE CAREFUL: We create remain address using our pk, wallet doesn't
+            -- show such addresses
+            (txAux, _) <- lift $ prepareMTx pm
+                                            epochSlots
+                                            getSigner
+                                            mempty
+                                            def
+                                            (NE.fromList allAddresses)
+                                            (map TxOutAux outputs)
+                                            curPk
+            txAux <$ (ExceptT $ try $ submitTxRaw diffusion txAux)
     case etx of
-        Left err -> logError $ sformat ("Error: "%stext) (toText $ displayException err)
-        Right tx -> logInfo $ sformat ("Submitted transaction: "%txaF) tx
+        Left err -> logError
+            $ sformat ("Error: " % stext) (toText $ displayException err)
+        Right tx -> logInfo $ sformat ("Submitted transaction: " % txaF) tx
```

Comment:
If this change was simply to get the max line width under 80 chars, i would suggest this is a perfect example of why the line width should stay at 100.

Reply:
I'm not sure that's a valid argument. The line width wasn't supposed to be 100 anyway, so this is just a change to conform to the style guide. Had the line been incorrectly as 102 when we wanted 100, the fact we have to change it isn't an argument for col width 102!